packages/libs/react-ui/src/components/Box/Box.tsx

```diff
@@ -9,66 +10,84 @@ export interface IBoxProps
     Pick<
       Sprinkles,
       | 'display'
+      | 'height'
```

Comment:
Not actually used in props below?

Reply:
ahh thank you for looking closely at these, I forgot to 😅